README.md

```diff
@@ -212,6 +212,63 @@ From here on you can follow the instructions available at the following sections
 
 All done!
 
+Install ansible-openwisp2 on docker container for development
+-------------------------------------------------------------
+
+If you want to contribute to `ansible-openwisp2` you should test changes in
```

Comment:
I think `you should run tests in your development environment to ensure your changes are not breaking anything.` is better